downstream/modules/eda/proc-modifying-memory-during-install.adoc

```diff
@@ -0,0 +1,11 @@
+[id="modifying-memory-during-install"]
+
+= Modifying the default memory limit for each rulebook activation during installation
+
+[role="_abstract"]
+By default, each rulebook activation container has a 200mb memory limit. You can modify this default value during installation by using the following procedure: .
+
+.Procedure
+. Navigate to the setup inventory file.
+. Add `automationedacontroller_podman_mem_limit` under [all:vars] section. For example, ________________. 
```

Comment:
Don't we need given an example here ?

Reply:
We can give an example like following:

```
automationedacontroller_podman_mem_limit='400m'

Reply:
Hi @jainnikhil30! I added that example. Can you take a look at the formatting?